content/kotlin/concepts/hashmaps/terms/remove.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.remove()'
+Description: 'Used to remove a key-value pair from a HashMap'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'HashMap'
+  - 'Data Types'
+  - 'Collections'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.remove()`** method is used to remove a key-value pair from a `HashMap` in Kotlin.
+
+## Syntax
+
+```pseudo
+fun <K, V> MutableMap<K, V>.remove(key: K): V?
+```
+
+It takes one argument:
```

Comment:
The bulleted list makes it appear both are arguments. Instead of introducing a bulleted list like this, just have two sentences: "It takes one parameter..." and "It returns..."